person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] I see that you have `attr_reader` for `id ` however there is no `id` variable in the constructor, kindly make sure to implement `id` inside the `initialize` method

Reply:
Thanks for the suggestion @Ismailco

Reply:
Fixed.